dev/TreeView/TreeViewNode.h

```diff
@@ -124,4 +124,4 @@ class TreeViewNodeVector :
     void Clear();
 };
 
-void throwIllegalMethodCallException();
\ No newline at end of file
+void throwIllegalMethodCallException();
```

Comment:
>throwIllegalMethodCallException [](start = 5, length = 31)

It's a weird pattern and you declared a same signature in two head files. It's better to move it to a shared project just like what we did for SharedHelpers. also put it into a namespace or class is a good practice

Reply:
Also methods should be PascalCased.